core/common/src/main/java/alluxio/exception/AlluxioException.java

```diff
@@ -20,7 +20,7 @@
  * the RPC framework and convert back without losing any necessary information.
  */
 @ThreadSafe
-public abstract class AlluxioException extends Exception {
+public class AlluxioException extends Exception {
```

Comment:
I made in concrete so that we can use it as a catch-all for any `RuntimeException` caught in the `RpcUtils#call` methods. I considered creating an `AlluxioRuntimeException` for this purpose, but decided it would be confusing since the exception would extend `AlluxioException`, not `RuntimeException`